.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: 'CI'
+# **What it does**: Runs test on the CI.
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
```

Comment:
**suggestion (optional)**

To use the same Node version in all CI jobs, I recommend creating an environment variable. You can find an example in [`ember-container-query`](https://github.com/ijlee2/ember-container-query/blob/2.1.0/.github/workflows/ci.yml#L12-L13).

Currently, the addon's `package.json` shows that it supports Node 10 and 12, contrary to 14 in CI. As you update the addon furthe and release new versions, I recommend keeping the Node version in `package.json` and the GitHub workflow up-to-date.

Reply:
Updated, I now keep it similar to the one already set in the original [travis.yml config](https://github.com/adopted-ember-addons/ember-launch-darkly/pull/318/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR11)